addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -670,10 +673,10 @@ const Wysiwyg = Widget.extend({
                 if (!this.ptp) return;
                 this.ptp.notifyAllClients('oe_history_step', historyStep, { transport: 'rtc' });
             },
-            onCollaborativeSelectionChange: _.throttle((collaborativeSelection) => {
+            onCollaborativeSelectionChange: debounce((collaborativeSelection) => {
                 if (!this.ptp) return;
                 this.ptp.notifyAllClients('oe_history_set_selection', collaborativeSelection, { transport: 'rtc' });
-            }, 50),
+            }),
```

Comment:
if you use debounce, you should keep the delay :-)
Check your diff to see if it is the only occurrence of this.

Reply:
I had already check ... but apparently ... not for this one. I make a second check ;)